docs/GettingStartedDocs/VisualStudioLinux.md

```diff
@@ -0,0 +1,132 @@
+  or [installation instructions for Ubuntu 18.04](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md), except that step 2 on those pages is outdated and result
+  in SGX not working.  Instead, replace step 2 with the instructions
+  [here](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/Contributors/SGX1GettingStarted.md)
+  prior to the Install section, which should work with either 16.04 or 18.04 even though
+  the page only mentions 16.04.
+
+Finally, configure Visual Studio with the address (or name) of your Linux build machine,
+via Tools -> Options -> Cross Platform -> Connection Manager -> Add. This step may take
+a minute or two, as Visual Studio will copy some files locally for use by IntelliSense.
+
+## Walkthrough: Creating a C/C++ Enclave Application
+
+We will now walk through the process of creating a C/C++ application that uses an enclave.
+
+1. Create a new Linux application using File -> New -> Project and find the Linux console
+   app template, which 
+   is called "Console App" (note: NOT the "Console App (.NET Core)") with the Linux
+   keyword.  (If it is not immediately visible, the template can be found under
+   Installed -> Visual C++ -> Cross Platform -> Linux.)
+   Give the project a name, LinuxApp for example.  This will create a "Hello World" console application.
+   Alternatively, if you already have such a Linux application using a Visual Studio project
+   file (.vcxproj file), you can start from your existing application.
+2. Configure the application project to use your Linux build environment, by right clicking
+   on the project in the Solution Explorer and selecting Properties -> Configuration
+   Properties -> General -> Remote Build Machine, and explicitly set it to the build machine
+   you configured in the Connection Manager.  (Due to a current Visual Studio bug, this step
+   is required even if the correct value is shown by default. In other words, make sure the
+   connection is shown in **bold**.)
+3. Also update Configuration Properties -> Debugging -> Remote
+   Debug Machine to your build machine, again due to a current Visual Studio 2019 bug.
+   At this point, you should be able to
+   build and debug the Hello World application. For further discussion, see the
+   [Linux debugging walkthrough](https://docs.microsoft.com/en-us/cpp/linux/deploy-run-and-debug-your-linux-project?view=vs-2019).
+4. Create an enclave library project by right clicking on the solution in the Solution Explorer
+   and selecting Add -> New Project -> Open Enclave TEE Project (Linux).  (If it is not
+   immediately visible, look under Installed -> Visual C++ -> Cross Platform -> Linux.)  Give it a name,
+   LinuxEnclave for example.  This will create a sample enclave with an ecall\_DoWorkInEnclave()
+   method exposed to applications, that will simply call an ocall\_DoWorkInHost() method that
```

Comment:
Consider using backticks for inline code so that ocall\_DoWorkInHost() renders as `ocall_DoWorkInHost()`.